index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class='head'> 
+            <div class="name">Yohannes Zemed </div>
```

Comment:
- [ ] Kindly use an `h1` tag for the main title (Yohannes Zemed ) to improve the readability and SEO of  your webpage.